backend/app/services/schwab_import.py

```
@@ -66,19 +66,43 @@ def map_schwab_transaction(txn: dict) -> dict | None:
-    if quantity > 0:
-        premium_per_share = abs(net_amount) / (quantity * 100)
+    # Premium per share: prefer the gross per-share price reported on the
+    # transferItem when Schwab provides it; otherwise gross-up the post-fee
+    # ``netAmount`` by adding the fee total back in before dividing. Schwab's
+    # ``netAmount`` field is the *net* dollar movement after commissions and
+    # exchange fees, so using it raw understates the premium by ``fees / (qty
+    # * 100)`` per share (issue #184: a 1-contract sell put at $0.30 with
+    # $0.66 in fees nets to $29.34, which rounds to $0.29 — the visible bug).
+    #
+    # Positive for sells (credits), negative for buys (debits).
+    # For sells, |netAmount| = gross − fees → add fees back to recover gross.
```

Ruff (0.15.12)

[warning] Comment contains ambiguous `−` (MINUS SIGN). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)